### DISCUSSION TITLE
Discrete Scroll for Mac OS 14.4.1

### DISCUSSION THREAD

**Discussion by Renvent** [community/community#120149] [Programming Help]
### Select Topic Area

Bug

### Body

This will not activate on my Mac. Reason given is the software needs to be updated for my version 14.4.1 (Sonoma) and to contact the developer. Is there a published version that works? I am attempting to correct an erratic third party mouse scrolling problem.

**Comment by ebndev** [comment]
Hey @Renvent,

Thanks for being a part of the GitHub community! 

Did you try opening an issue to contact the developer? The README on the repo says 14.4.1 is not yet supported yet but maybe opening an issue will bring it to their attention. Best of luck!
